config/database.yml

```
@@ -1,22 +1,21 @@
 default: &default
   adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  host: <%= ENV["DATABASE_HOST"] %>
+  username: <%= ENV["DATABASE_USERNAME"] %>
+  password: <%= ENV["DATABASE_PASSWORD"] %>
```

Maybe more prettier is using fetch, and clean .env.example file
```suggestion
  host: <%= ENV.fetch("DATABASE_HOST", "postgres") %>
  username: <%= ENV.fetch("DATABASE_USERNAME", "postgres") %>
  password: <%= ENV.fetch("DATABASE_PASSWORD", "postgres") %>
```

---

Or use DATABASE_URL